app/controllers/api/databases.php

```diff
@@ -1615,8 +1615,55 @@ function updateAttribute(
         Event $events
     ) {
         $key ??= $relatedCollectionId;
+        $twoWayKeyNull = is_null($twoWayKey);
         $twoWayKey ??= $collectionId;
 
+        $databaseDocument = Authorization::skip(fn() => $dbForProject->getDocument('databases', $databaseId));
```

Comment:
Can we do this in the `createAttribute` function, I think we have a lot of this code there already